lib/seattleflu/db/cli/command/clinical.py

```diff
@@ -102,9 +108,7 @@ def uw_clinical(uw_filename, uw_nwh_file, hmc_sch_file, output):
     clinical_records['individual'] = clinical_records['individual'].apply(generate_hash)
     clinical_records['identifier'] = clinical_records['identifier'].apply(generate_hash)
 
-    session = DatabaseSession()
-    with session, session.cursor() as cursor:
-        clinical_records.apply(lambda x: insert_clinical(x, cursor), axis=1)
+    print(clinical_records.to_csv(index=False))
```

Comment:
Do we want to match the manifest command and make this print out ndjson?

Reply:
I had a similar thought. My reason for choosing csv over ndjson comes down to an argument for splitting parse & upload Thomas laid out in the related Trello card:
>better inspectability of parsed records (you can parse them without uploading anywhere)

I figured if the original data were in CSV format and someone might want to inspect them after parsing, it would be easier for them to do so if the output was in CSV format, too.

Of course, Thomas is taking manifests from XLSX format and outputting them to ndjson. However, I don't think manifest data are really meant to be read in the same way clinical data are. Perhaps I'm just splitting hairs here, but I appreciate your comment and welcome any feedback.

Reply:
The ability to inspect the output of parse is an important consideration. :-) From my perspective, NDJSON is just as inspectable as CSV, but I realize that is not the case for everyone!

I think the things that swayed and sway my thinking towards NDJSON are:

1. It matches the exact format/document which ends up in the database and doesn't go through an additional, harder-to-inspect layer of transformation by `pd.read_csv` and `df.to_json`. This makes the data more inspectable in some ways (although less accessible as NDJSON, maybe?).

2. It is clear how to represent nested data structures, like the provenance information. This kind of information isn't in the clinical data pipeline yet, but I can imagine wanting to add it sooner than later.

I'm not dead set on NDJSON here or 100% against the current CSV output, so would like to hear any thoughts you both have.

Reply:
It sounds like it's two against one :anguished: . 
I think I prefer consistency in the parse/upload process over my preference for reading CSV vs. JSON. I'm working on the changes now.